Observer/AdyenHppDataAssignObserver.php

```diff
@@ -39,7 +39,10 @@ class AdyenHppDataAssignObserver extends AbstractDataAssignObserver
     const DF_VALUE = 'df_value';
     const SSN = 'ssn';
 	const OWNERNAME = 'ownerName';
+    const BANKACCOUNTOWNERNAME = 'bankAccountOwnerName';
```

Comment:
Can you add the underscore to make it more readable for all of these constants ? So BANKACCOUNT_OWNER_NAME BANKACCOUNT_NUMBER, BANK_LOCATION_ID.

Reply:
indeed